index.html

```diff
@@ -10,16 +10,49 @@
     <link href="https://fonts.googleapis.com/css2?family=Comic+Neue:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap" rel="stylesheet">
 </head>
 <body>
-    <h1>Awesome-books</h1>
-    <table id="list-books">
+    <header class="header">
+        <a href="#list" id="logo">Awesome Books</a>
+        <nav id="navbar">
+            <ul id="contain-nav-item">
+                <li id="nav-list" class="nav-item">
+                    <a id="nav-list-a" class="nav-item-a" href="#list">List</a>
+                </li>
+                <li id="nav-add-new" class="nav-item">
+                    <a id="nav-add-new-a" class="nav-item-a" href="#add-new">Add new</a>
+                </li>
+                <li id="nav-contact" class="nav-item">
+                    <a id="nav-contact-a" class="nav-item-a" href="#contac">Contact</a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+     <section id="list" class="list">
+        <h1>Awesome-books</h1>
+        <table id="list-books">
+      
+        </table>
+     </section>      
+     <section id="add-new" class="hiden add-new">
+        <h2>Add a new book</h2>
+        <form id="form" action="">
+            <input type="text" id="title"  class="from-control" name="title" placeholder="Title" required>
+            <input type="text" id="author"  name="author" placeholder="Author" required>
+            <button type="submit" id="add" placeholder="Add">Add</button>
+        </form>
```

Comment:
[ OPTIONAL ]
Good work implementing your form that successfully adds books to the `List`. However, there is a tiny suggestion I would want to make. ♻️ 

- [ ] Kindly clear your form after submitting to help your users know when submitting is successful. This is a good User experience practice for your development. ⬆️ 

You can use the code below as a guide 🔽 
`document.getElementById("myForm").reset();`